CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntaxbuilder/BuildableNodesFile.swift

```diff
@@ -82,9 +87,11 @@ private func createConvenienceInitializer(node: Node) -> InitializerDeclSyntax?
         }
       }
       builderParameters.append(FunctionParameterSyntax(
-        "@\(builderInitializableType.resultBuilderBaseName) \(child.swiftName)Builder: () -> \(builderInitializableType.syntax) = \(defaultArgument)",
+        "@\(builderInitializableType.resultBuilderBaseName) \(child.swiftName)Builder: () \(addRethrows ? "throws" : "") -> \(builderInitializableType.syntax) = \(defaultArgument)",
         for: .functionParameters
       ))
+
+      willBuilderThrow = true
```

Comment:
I don't think we need this variable. AFAICT it always has the same value as `shouldCreateInitializer`. And if we get to the point where we're checking the variable, we know that `shouldCreateInitializer` is true because there's a `guard`.